features.tex

```diff
@@ -0,0 +1,193 @@
+\section{Functionality in future DM releases}
+
+This is currently not an exhaustive feature list but gives an indication at a high level of the features in each release which will be verified by the corresponding verification test campaign.
+As the test plans are written this will become a list of requirements to be tested for that release and thus begin o fill out the verification control database (currently to be in Jira).
+
+In the feature lists below the internal milestone for that feature is given in parenthesis)
+
+
+Each section here is a test milestone from \citeds{LDM-503} - the same labels are used. The timeline is in the DM schedule using the same labels and depicted in \figref{fig:schedule}
+
+\begin{figure}[htbp]
+        \begin{center}
+                 \includegraphics[width=\textwidth]{images/DMMasterSchedule}
+                 \caption{DM major milestones (LDM-503-$x$) in the LSST schedule. \label{fig:schedule}}
+         \end{center}
+ \end{figure}
+~
+
+\subsection{Science Platform with WISE data in PDAC \textbf{(LDM-503-1)}\label{LDM-503-1}}
```

Comment:
Many of the milestones given below don't seem to correspond to features in software releases. For example:

- Networks are presumably not tagged and released as software products(!)
- Services at NCSA are clearly software backed. Is the expectation that the software used to run each service is tagged and released with the rest of the DM products?